src/extension.ts

```diff
@@ -656,6 +656,14 @@ function launchMetals(
       });
     });
 
+    registerCommand("metals.toggle-implicit-conversions-and-classes", () => {
```

Comment:
I'm trying to think if this will be used by any other editors that may use `metals-languageclient`. Do you think we should add this there, or is this probably pretty VS Cod specfic?

Reply:
I think currently it's pretty much VS Code specific, it could be useful in cases that we show decorations, but emacs does not used the language client

Reply:
For hovers, it will just start working with the next hover.